src/views/PointOfSale/POSEditView.vue

```diff
@@ -6,34 +6,38 @@
     <div class="content-wrapper">
       <div class="pos-row">
         <div class="pos-general-info">
-          <h3>{{ $t("c_POSCreate.General") }}</h3>
-          <span class="general-info-block">
-            <b>{{ $t("c_POSCreate.Title") }}</b>
-            <InputText class="input" type="text" v-model="title"/>
-          </span>
-          <div class="general-info-block">
-            <b>{{ $t("c_POSCreate.Owner") }}</b>
-            <p>{{ posDisplayName }}</p></div>
-          <div>
-            <span class="general-info-block" style="flex-direction: row;">
-              <Checkbox v-model="useAuthentication"
+          <form @submit="handleEditPOS">
+            <h3>{{ $t("c_POSCreate.General") }}</h3>
+            <span class="general-info-block">
+              <b>{{ $t("c_POSCreate.Title") }}</b>
+              <InputText class="input" type="text" v-bind="title"/>
+              <span class="error-text">{{ errors.title }}</span>
+            </span>
+            <div class="general-info-block">
+              <b>{{ $t("c_POSCreate.Owner") }}</b>
+              <p>{{ posDisplayName }}</p>
+            </div>
+            <div>
+            <span class="general-info-block" style="flex-direction: row; align-items: center;">
+              <Checkbox v-bind="useAuthentication"
                         inputId="useAuthentication"
                         name="useAuthentication"
                         value="useAuthentication"
                         :binary="true"
               />
               <label for="useAuthentication">{{ $t("c_POSCreate.Use authentication") }}</label>
             </span>
-          </div>
-          <div class="general-info-block">
-            <b>{{ $t("c_POSCreate.Selected containers") }}</b>
-            <ul class="selected-containers">
-              <li v-for="container in selectedContainers" :key="container.id">
-                {{ container.name }}
-              </li>
-            </ul>
-          </div>
-          <Button id="create-pos-button" :label="$t('c_POSCreate.Edit')" @click="updatePointOfSale" severity="success"/>
+            </div>
+            <div class="general-info-block">
+              <b>{{ $t("c_POSCreate.Selected containers") }}</b>
+              <ul class="selected-containers">
+                <li v-for="container in selectedContainers" :key="container.id">
+                  {{ container.name }}
+                </li>
+              </ul>
+            </div>
+            <Button id="create-pos-button" :label="$t('c_containerEditModal.save')" type="submit" severity="success"/>
```

Comment:
Why is this button not in the top right on the same line as the edit point of sale :(

Why does pressing it give no user feedback? did it work? did it not? dit it error :()

Reply:
Already marked as #18 in the code.